vuepress/docs/next/docs/reference/glossary.md

```diff
@@ -0,0 +1,53 @@
+| Entando Cloud Hub | A SaaS instance of an Entando Hub that contains a public and private collection of components |
+| Entando Cluster | The infrastructure Entando deploys on Kubernetes via the Entando Operator and controllers; each element is associated with a custom resource definition, if applicable |
+| Entando Component Generator (ECG) | Entando's implementation of a JHipster blueprint that generates the components used to build an Entando Application via automation and templating |
+| Entando Component Manager (ECM) | Provides functionality to build and organize micro frontends and widgets using the App Builder and manages the connections between an application and the installed microservices |
+| Entando Content | A structured element representing a set of information built using Entando Content Attributes and added to an Entando Application via the Entando App Builder |
+| Entando Content Type | The structure of specific content, defined by a set of content attributes |
+| Entando Content Attribute | The basic data that define a content type |
+| Entando Content Template | The style or layout of a content type where a single content type can have multiple content models defining different ways to render the same content |
+| Entando Digital Asset | An image, document, or other media file that is uniquely identifiable and stored digitally in a format supported by the Entando Platform |
+| Entando Hub |  A repository (local, remote, public, or private) containing components built with the Entando Platform; A single Entando App Builder can connect to 1 or more Entando Hubs |
+| Entando Identity Management | Entando's Keycloak- and token-based user management and authentication system |
+| Entando Kubernetes integration service (k8s-service) | A function of the Entando Cluster infrastructure custom resource that provides an abstraction layer between Entando microservices and the APIs exposed by Kubernetes |
+| Entando Local Hub | The local component repository in an Entando App Builder used to store and retrieve shareable components for reuse within an Entando Application |
+| Entando Operator | Provides installation and application lifecycle automation for Entando Applications, microservices and required infrastructure services, e.g. databases and Keycloak |
+| Entando Platform | The leading open source application composition platform for development with a Kubernetes architecture |
+| Entando Platform Capability (EPC) | A packaged capability that adds functionality to the platform and/or additional UX controls to the App Builder |
+| Entando Plugin | A microservice that exposes APIs reusable by one or more Entando Applications |
+| Entando UX Fragment | An HTML block containing Freemarker tags that allow content to be rendered dynamically according to context |
+| Entando Web Content Management System (WCMS) | Entando's lightweight content and digital asset management system that supports headless operations |
```

Comment:
Let's remove "that supports headless operations". It has APIs but we don't want to compete with Strapi when we announce it, and our impl is rarely used headless.